app/views/admin/groups/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<% content_for :page_title, [t('hyku.admin.groups.title.new'), t('hyku.admin.title'), application_name].join(' // ') %>
+<h1>
+  <span class="fa fa-users"></span>
+  <%= t('hyku.admin.groups.title.new') %>
+</h1>
+<div class="panel-tab-wrapper">
+  <ul class="nav nav-tabs">
+    <li role="presentation" class="active">
+      <a href="javascript:void(0);"><%= t('hyku.admin.groups.nav.attributes') %></a>
```

Comment:
In `app/views/admin/groups/new.html.erb` the tab anchors are disabled because you can neither add members to nor remove a group that doesn’t exist yet.